lib/content-services/src/lib/content-node-selector/content-node-dialog.service.ts

```diff
@@ -179,7 +179,7 @@ export class ContentNodeDialogService {
         });
 
         const data: ContentNodeSelectorComponentData = {
-            title: this.getTitleTranslation(action, contentEntry.name),
+            title: this.getTitleTranslation(action, this.translation.instant('DROPDOWN.MY_FILES_OPTION')),
```

Comment:
Are you sure this doesn't break the functionality of the dropdown in ADW on Copy / Move action?